Packs/CommonScripts/Scripts/ShowLocationOnMap/README.md

```diff
@@ -0,0 +1,23 @@
+Displays a geographical location in the indicator layout (the **geolocation** field).
```

Comment:
Using the geolocation field... the location you will see in the layout (in the chosen dynamic section)

maybe we should add a reference to the article you are working on 
https://github.com/demisto/content-docs/pull/919#issuecomment-1008256723

Reply:
Yes I agree. I thought we can't reference it yet until the article is pushed to the master?

Reply:
True, :) So the question is - if should we merge it now and add the reference later? Or wait until we merge the article?

Reply:
I think we should probably wait until we merge the article. Also this article is not validating - is that because we don't have approvals yet?